src/verinfast/system/sysinfo.py

```diff
@@ -0,0 +1,71 @@
+import json
+import subprocess
+from typing import Dict, Any
+import platform
+import os
+
+
+def get_system_info() -> Dict[str, Any]:
+    """
+    Collects system information using hyfetch and
+    returns it as a JSON-compatible dictionary.
+    Falls back to basic system info if hyfetch fails.
+    """
+    try:
+        # Run hyfetch with a 5 second timeout
```

Comment:
prefer importing hyfetch directly, it's a python package. this kind of run command doesn't work with pipx installs.